homeassistant/config_entries.py

```diff
@@ -828,6 +843,47 @@ async def _async_init_reauth(
             translation_key="config_entry_reauth",
         )
 
+    @callback
+    def async_start_reconfigure(
+        self,
+        hass: HomeAssistant,
+        context: dict[str, Any] | None = None,
+        data: dict[str, Any] | None = None,
+    ) -> None:
+        """Start a reconfigure flow."""
+        # We will check this again in the task when we hold the lock,
+        # but we also check it now to try to avoid creating the task.
+        if any(self.async_get_active_flows(hass, {SOURCE_RECONFIGURE})):
```

Comment:
```suggestion
        if self.async_get_active_flows(hass, {SOURCE_RECONFIGURE}):
```

Unless I'm missing something, this looks like a useless `any` expression

Reply:
Ahh. its a generator....